miqa/core/rest/global_settings.py

```diff
@@ -25,7 +25,7 @@ def get_object(self):
         return GlobalSettings.load()
 
     def get_permissions(self):
-        if self.request.method == 'GET':
+        if self.request.method != 'PUT':
```

Comment:
It seems safer to explicitly enumerate the conditions under which we are more permissive.

Reply:
Good point, changed in [bd32b0b](https://github.com/OpenImaging/miqa/pull/478/commits/bd32b0b660dc8fde2237726198aa787c6b40b09e)